mobile/lib/features/trade/buy_sell_bottom_sheet.dart

```diff
@@ -37,25 +121,72 @@ class BuySellContents extends StatelessWidget {
     String label = direction == Direction.buy ? "Buy" : "Sell";
     Color color = direction == Direction.buy ? tradeTheme.buy : tradeTheme.sell;
 
-    return Container(
-      padding: const EdgeInsets.only(top: 20),
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        children: [
-          Center(child: Text("$label Content")),
-          const Spacer(),
-          Column(
-            crossAxisAlignment: CrossAxisAlignment.stretch,
-            children: [
-              ElevatedButton(
-                  onPressed: () {},
-                  style: ElevatedButton.styleFrom(
-                      backgroundColor: color, minimumSize: const Size.fromHeight(50)),
-                  child: Text(label)),
-            ],
-          )
-        ],
-      ),
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        Wrap(
+          runSpacing: 15,
+          children: [
+            Row(
+              children: const [
+                Flexible(child: Text("Available Balance:")),
+                Flexible(child: Text("2,000 sats"))
+              ],
+            ),
+            TextFormField(
+              controller: TextEditingController()..text = '19,900.0',
+              enabled: false,
+              decoration: const InputDecoration(
+                border: OutlineInputBorder(),
+                hintText: "Price",
+                labelText: "Market Price",
```

Comment:
Sorry, I am not sure what you are referring to - you mean pull the text into localization files so that all app text can be easily translated into other languages?